portal-kernel/src/com/liferay/exportimport/kernel/exception/LARScopeException.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.exportimport.kernel.exception;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author Alberto Javier Moreno Lage
+ */
+public class LARScopeException extends PortalException {
```

Comment:
I would not add this to portal-kernel. It seems we are only using it in the export-import-service module, so I would move it there and not expose it to any other modules for now.